cfme/tests/infrastructure/test_vmware_provider.py

```diff
@@ -163,7 +164,11 @@ def test_vmware_reconfigure_vm_controller_type():
             3.Reconfigure VM opion should be enabled
             4.Controller type should be listed
     """
-    pass
+    vms_collections = appliance.collections.infra_vms
+    vm = vms_collections.instantiate(name='cu-24x7', provider=provider)
+    view = navigate_to(vm, 'Reconfigure')
+    row = view.disks_table[0]
```

Comment:
A comment here should be good saying that you are grabbing the first row in the table.